app/src/api/services/dataExtract.ts

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+
+import db from '../../db';
+
+interface DataExtractRow {
+    extract_id: number;
+    extracted_on: Date;
+    extract_path: string;
+}
+
+interface DataExtract {
+    extract_id: number;
+    extracted_on: Date;
+    download_path: string;
+}
+
+export async function listDataExtracts(): Promise<DataExtract[]> {
```

Comment:
Here and elsewhere - let's follow the convention of exporting at the bottom of a file in one place.